store/views.py

```
@@ -54,7 +55,12 @@ class CustomerViewSet(ModelViewSet):
 
     @action(detail=False, methods=['GET', 'PUT'], permission_classes=[IsAuthenticated])
     def me(self, request):
-        customer = Customer.objects.get(user_id=request.user.id)
+
+        try:
+            customer = Customer.objects.only('id').get(user_id=request.user.id).id
+        except Customer.DoesNotExist:
+            raise NotFound(detail="Customer not found for the current user.")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)